src/Components/PaperEditor/TranscriptsContainer/TranscriptMenu/index.js

```diff
@@ -8,70 +8,62 @@ import ButtonGroup from 'react-bootstrap/ButtonGroup';
 import DropdownButton from 'react-bootstrap/DropdownButton';
 import LabelsList from '../LabelsList/index.js';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHighlighter, faCog } from '@fortawesome/free-solid-svg-icons';
+import { faCog, faTag } from '@fortawesome/free-solid-svg-icons';
+import './index.scss';
 
 const TranscriptMenu = (props) => {
-  const labels = props.labels;
+  const { labels, activeLabel } = props;
 
-  const HighlightButton = (
+  useEffect(() => {
```

Comment:
Nice, this totally fixes the initial colour flashes on load :)

Reply:
Do you need `props` in that dependency array at the end? Don't see it being used in this hook